README.md

```diff
@@ -1,28 +1,29 @@
-# The Topology Builder for Apache Kafka (Automation, Gitops, SelfService)
+# An operational manager for Apache Kafka (Automation, Gitops, SelfService)
+=======
 
 <a href="https://codeclimate.com/github/purbon/kafka-topology-builder/maintainability"><img src="https://api.codeclimate.com/v1/badges/ef4bcda7d1b5fd0a4f1e/maintainability" /></a> ![CI tests](https://github.com/kafka-ops/kafka-topology-builder/workflows/CI%20tests/badge.svg?branch=master) [![Gitter](https://badges.gitter.im/kafka-topology-builder/community.svg)](https://gitter.im/kafka-topology-builder/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![Documentation Status](https://readthedocs.org/projects/kafka-topology-builder/badge/?version=latest)](https://kafka-topology-builder.readthedocs.io/?badge=latest)
 
-Topology Builder for Apache Kafka helps you automate the management of your things within Apache Kafka, from Topics, Configuration to Metadata but as well
-Access Control, Schemas. 
+Julie Ops helps you automate the management of your things within Apache Kafka, from Topics,
+Configuration to Metadata but as well Access Control, Schemas. 
 More items are plan, check [here](https://github.com/kafka-ops/kafka-topology-builder/issues) for details. 
 
 ## The motivation 
 
-A typical question many teams face when building, or growing, an Apache Kafka infrastructure is how to manage
-topics, configuration, metadata, ACLs, etc in a more automated way to provide their teams with the most autonomy 
-possible while reducing the time involved in this administrative tasks. 
+One of the typical questions while building an Apache Kafka infrastructure is how to handle topics, 
+configurations and the required permissions to use them (Access Control List).
 
-The Topology Builder for Apache Kafka, in close collaboration with Git and CI/CD, (Jenkins, etc) is here to help you build it.
+The Julie Ops cli, in close collaboration with git and Jenkins (CI/CD) is here to help you setup an 
+organised and automated way of managing your Kafka Cluster.
  
 ## Where's the docs?
 
 We recommend taking time to [read the docs](https://kafka-topology-builder.readthedocs.io/).
 There's quite a bit of detailed information about Gitops, Apache Kafka and how this project can help you automate 
 the common operational tasks.
 
-## Automating Management with CI/CD and Gitops
+## Automating Management with CI/CD and gitops
```

Comment:
nit: main title uses "Gitops" and here "gitops". Not sure what's the best naming here, "GitOps" perhaps?